File: dart/scripts/hello_loop.dart
void main() {
    var worldList = ['a', 'b', 'c'];

    worldList.forEach((String world) {
        print('world is $world!');
    });

    var worldMap = {
      'a': 'aaa',
      'b': 'bbb',
      'c': 'ccc',
    };

    worldMap.forEach((key,value) {
        print('world $key is $value!');
    });
}

File: nodejs/scripts/helloLoop.js
const worldList = ['a', 'b', 'c']

for (const world of worldList) {
  console.log(`world is ${world}!`)
}

const worldObject = {
  a: 'aaa',
  b: 'bbb',
  c: 'ccc'
}

for (const [key, value] of Object.entries(worldObject)) {
  console.log(`world ${key} is ${value}!`)
}
